go/pkg/client/tree_test.go

```diff
@@ -44,6 +48,16 @@ func mustMarshal(p proto.Message) []byte {
 	return b
 }
 
+func prependExecRootPlaceholder(target string) string {
+	return fmt.Sprintf("%s/%s", execRootPlaceholder, target)
```

Comment:
use `filepath.Join` or `path.Join`?

Reply:
This function is used in `SymlinkNode`, where the API says the target path is always separated by `/`: https://github.com/bazelbuild/remote-apis/blob/aa8e718768c2973274c2a23e9dcc27b42afea794/build/bazel/remote/execution/v2/remote_execution.proto#L798-L807

That said, I noticed that the existing tests's `inputPath` are also constructed with `/`, instead of `filepath.Join`. I wonder why this is not a problem on Windows..

Reply:
I think this repo doesn't have Windows coverage.

Reply:
hmm, i thought `cas` already worked on Win?

Reply:
This repo works on Windows -- there are some tests that are Windows or Linux specific (see e.g. cache_posix_test.go https://github.com/bazelbuild/remote-apis-sdks/blob/master/go/pkg/filemetadata/BUILD.bazel), but this test should work on both.